src/scripts/main.js

```diff
@@ -357,4 +357,35 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const dashboardTable = document.querySelector('.dashboard');
```

Comment:
Ensure that the '.dashboard' element exists in the DOM before attempting to append rows to it. Consider adding a null check for 'dashboardTable' to prevent potential runtime errors.